examples/scroll.py

```diff
@@ -151,25 +152,29 @@ def main(image_file=None):
         clock.tick()
 
         going = True
+
         while going:
             # wait for events before doing anything.
             # events = [pg.event.wait()] + pg.event.get()
             events = pg.event.get()
 
+            # During the loop, if a key is held, scroll the view.
```

Comment:
I often mostly tell people to avoid using key.get_pressed and just use the event loop. But I wonder what other people think? I'm also not sure about just using if, and not elif.